test/unit/rules/no-named-arguments-with-global-attribute-names-test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const { makeError } = require('../../../lib/rules/no-named-arguments-with-global-attribute-names');
+
+const good = [
+  '{{my-component class="shaded"}}',
+  '{{#my-component class="shaded"}}welcome{{/my-component}}',
+  '<MyComponent class="shaded" />',
+  '<MyComponent class="shaded">bienvenidos</MyComponent>',
+  '<MyComponent aria-valuemin={{0}} />',
+  '<MyComponent data-id="123" />',
+  '<MyComponent onclick={{action "myAction"}} />',
+  '<MyComponent {{on "click" this.myAction}} />',
+  '{{log "foo" class="bar"}}',
+];
+
+const bad = [
+  {
+    template: '<MyComponent @class="shaded" />',
+    result: {
+      message: makeError('@class'),
+      source: '<MyComponent @class="shaded" />',
+      line: 1,
+      column: 0,
+    },
+  },
+  {
+    template: '<MyComponent @aria-valuemin={{1}} />',
```

Comment:
How is this different from line 12, other than the number inside curly braces?

Reply:
The leading `@`